### ISSUE TITLE
Installation from npm does not work

### ISSUE THREAD

**Issue by JirkaChadima** [windingtree/wt-write-api#76]
1. Run `npm i -g @windingtree/wt-write-api && WT_CONFIG=playground wt-write-api`
2. Watch the startup command fail epically with

```
TypeError: p.a is not a constructor
    at new t (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/node_modules/@windingtree/wt-js-libs/dist/node/wt-js-libs.js:1:2532)
    at Function.value (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/node_modules/@windingtree/wt-js-libs/dist/node/wt-js-libs.js:1:2611)
    at new t (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/node_modules/@windingtree/wt-js-libs/dist/node/wt-js-libs.js:1:37538)
    at Function.value (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/node_modules/@windingtree/wt-js-libs/dist/node/wt-js-libs.js:1:37802)
    at new t (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/node_modules/@windingtree/wt-js-libs/dist/node/wt-js-libs.js:1:40494)
    at Function.value (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/node_modules/@windingtree/wt-js-libs/dist/node/wt-js-libs.js:1:40649)
    at Object.<anonymous> (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/src/config/playground.js:9:20)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
```

Apparently it's an issue with the compiled wt-js-libs, so it's probably a bug in wt-js-libs. What's weird is that it seems to be working just fine within the docker image.

Related to windingtree/wt-write-api#206

**Comment by JirkaChadima** [comment]
Hopefully fixed with #78

**Comment by JirkaChadima** [comment]
The problem is that package-lock is not distributed via npm and web3 goes haywire with package updates. Because the guys there don't give a shit about semantic versioning, it got broken. Pinning the web3 dependencies to a working version should temporarily solve this problem.
